### ISSUE TITLE
ValueError when try exporter

### ISSUE THREAD

**Issue by PhilippeAccorsi** [danihodovic/borg-exporter#60]
Hi,

I have installed the exporter but when I execute `borg-exporter enable-systemd` I see this error:
```
root@*********:/tmp# borg-exporter enable-systemd
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
cli.py 1 <module>
<frozen importlib._bootstrap> 1007 _find_and_load
<frozen importlib._bootstrap> 986 _find_and_load_unlocked
<frozen importlib._bootstrap> 680 _load_unlocked
pyimod02_importers.py 419 exec_module
cli.py 52 <module>
cli.py 17 config_opt
decorators.py 373 decorator
core.py 2536 __init__
core.py 2151 __init__
ValueError:
'default' must be a list when 'multiple' is true.
[129862] Failed to execute script 'cli' due to unhandled exception!
```

Any idea where I need to change something ?

In `/etc/borgmatic/config.yaml` I have just set:
```
location:
    source_directories:
        - /data/nfs
    repositories:
        - ssh://******:***/backup/borgbackup-repository
storage:
    encryption_passphrase: "*********"
retention:
    keep_hourly: 24
    keep_daily: 7
    keep_weekly: 4
    keep_monthly: 4
```

If I execute `borgmatic --info` I have this result
```
root@*****:~# borgmatic --info
usage: borgmatic [-h] [-c [CONFIG_PATHS ...]] [--excludes EXCLUDES_FILENAME] [-n] [-nc] [-v {-1,0,1,2}] [--syslog-verbosity {-1,0,1,2}] [--log-file-verbosity {-1,0,1,2}]
                 [--monitoring-verbosity {-1,0,1,2}] [--log-file LOG_FILE] [--override SECTION.OPTION=VALUE [SECTION.OPTION=VALUE ...]] [--no-environment-interpolation] [--bash-completion] [--version]
                 ...
borgmatic: error: unrecognized arguments: --info
Error parsing arguments: /usr/bin/borgmatic --info

Need some help? https://torsion.org/borgmatic/#issues
```

I have installed borgmatic from source on Debian 12, so borgmatic version is 1.7.7
Do you think it is a problem with my version ? (borgbackup is in 1.2.4 )

**Comment by danihodovic** [comment]
Should be fixed in #61. I've re-tagged and uploaded latest.

**Comment by PhilippeAccorsi** [comment]
Thanks for the fix \o/
